submitted_models/ctu_cras_norlab_spot_sensor_config_3/model.sdf

```diff
@@ -0,0 +1,2095 @@
+<?xml version='1.0' encoding='utf-8'?>
+<sdf version="1.7">
+  <!-- CAUTION: This is an autogenerated file, DO NOT EDIT IT!
+  It was generated by script update_robot_sdf and is based on urdf/spot.urdf.xacro source file.
+  Do all modifications there and then run the update script to regenerate the SDF. -->
+  <model name="ctu_cras_norlab_spot_sensor_config_1">
+    <link name="base_link">
+      <inertial>
+        <mass>16</mass>
+        <inertia>
+          <ixx>0.0973333</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>1.02467</iyy>
+          <iyz>0</iyz>
+          <izz>1.04493</izz>
+        </inertia>
+      </inertial>
+      <collision name="base_link_collision">
+        <geometry>
+          <mesh>
+            <uri>../bosdyn_spot/meshes/body_collision.stl</uri>
```

Comment:
I think we should change this to a Fuel URI once #841 is merged and the Spot model is uploaded to Fuel. Relative paths like this should really only be used to reference items inside the model's own directory.

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d